dataeng/resources/load-json-events-to-s3.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY*2 ))
+fi
```

Comment:
Interesting that this is here and not in the other job.   I don't think it really matters, though, for these incremental jobs.   It's one reducer per day, and we're running one day.    (So the non-zero number of instances is helping only with the mapping phase.)

Reply:
Oh, interesting.  Apart from this line, the two bash scripts are identical.  You could just get rid of this script altogether, best I can tell, and have both jobs use load-events-to-s3.sh.